extensions/sql-migration/src/service/contracts.ts

```diff
@@ -0,0 +1,553 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RequestType, NotificationType } from 'vscode-languageclient';
+import * as azdata from 'azdata';
+
+export interface IMessage {
+	jsonrpc: string;
+}
+
+export interface SqlMigrationAssessmentParams {
```

Comment:
You should clean up the stuff from mssql too since that shouldn't be needed anymore

Reply:
That's supposed to be a work item for the migration team. Cleaning up the mssql contracts will break the current version of the extension in insiders build.

Reply:
It won't break the existing published function - typing is all lost at runtime and const enums are inserted directly into the code at build time. 

(the mssql.d.ts file doesn't even exist at runtime. That's just a compile time thing)

Reply:
Synced up offline about this. The migration service feature currently lies in MSSQL which will be needed for the current version of the extension function.